### ISSUE TITLE
Issue regarding \n in level5.jsh

### ISSUE THREAD

**Issue by JustinGohkh** [nus-cs2030/2223-s2#155]
Hi, I was unable to use \n in my accumulation of a result string in level5.jsh. The output was correct but it did not pass the test cases as the \n got concatenated into the string instead of giving a new line. I worked around this by printing out my output instead of returning it, but I would still like to know why the \n didn't work in the JSH file? or is it a jshell issue? Thanks!

**Comment by Xequil1** [comment]
Hey, I am running into the same issue.
Just a question though, wouldn't printing the output be flagged by the check style?

**Comment by JustinGohkh** [comment]
Nope, just set the return type as void for findBestBooking() and use System.out.println(res) for res as my accumulated String and it worked for me.

**Comment by JustinGohkh** [comment]
It didn't trigger the checkstyle in my case

**Comment by imkwokyong** [comment]
Yep, I used System.out.println() too.

**Comment by e0959170** [comment]
for this lab i think we can js directly print multiple lines (i.e. multiple output is allowed)

**Comment by haqanh** [comment]
I ran into the issue too. Am a bit confused, does this mean we are supposed to return multiple outputs repeatedly?

**Comment by Xequil1** [comment]
> I ran into the issue too. Am a bit confused, does this mean we are supposed to return multiple outputs repeatedly?

No your function should return nothing or not have a return at all.

Recall that we returned a String for the other labs, in this case we have void type in place of String which means the return value is null.

Instead of concatenating each string to some string for you to return, what you would want to do is to print the string out instead just as how the other users in this thread did it.

**Comment by lershuan** [comment]
> Nope, just set the return type as void for findBestBooking() and use System.out.println(res) for res as my accumulated String and it worked for me.

How do you return void? I can't seem to "return;"

**Comment by taugwa** [comment]
> > Nope, just set the return type as void for findBestBooking() and use System.out.println(res) for res as my accumulated String and it worked for me.
> 
> How do you return void? I can't seem to "return;"

don't think you need a return statement at all for a void method

**Comment by leonardwx** [comment]
i faced the similar issue before and what i did was to use both System.out.printIn(bookings) and System.out.printf("\n")!
